### ISSUE TITLE
Error Registering Routines

### ISSUE THREAD

**Issue by andyvanhelsing** [actions-on-google/actions-on-google-nodejs#302]
Not sure if this is a client library or the Google Assistant Action:

When a user tries to set up a routine the app unexpectedly exits when trying to register the routine with the following message appears in the Google Assistant Action log:

> MalformedResponse at expected_inputs[0].possible_intents[0].input_value_data: Not allowed to register updates for Routines for non-preview app

This error does not occur when I have testing enabled but is consistently reproduced on the live version of the app.

I am using the standard syntax to register the routine:
```
conv.ask(new RegisterUpdate({
intent: `[INTENT_NAME]`,
frequency: `ROUTINES`,
}))
```

**Comment by Canain** [comment]
Hi, what locale are you using for your action? It may not be available yet for public distribution for your locale.

**Comment by andyvanhelsing** [comment]
I am in en/us

**Comment by andreask1** [comment]
I have the same issue (also en/us). It's working in test mode, but when deploying to the alpha channel the error mentioned above appears in the logs. However, it's possible to add routines with the "Add" button on the action page even when using the alpha channel.

**Comment by andyvanhelsing** [comment]
That is a perfect description of what I've experienced. Have you been in contact with anyone at Google about this?

**Comment by andreask1** [comment]
No, I just saw the error appearing in the alpha channel and this issue is the only result that comes up when googling the error message from the logs. I'm not sure but I think it worked a couple of days ago when I first pushed it to alpha.
